circle.yml

```diff
@@ -46,7 +46,7 @@ test:
         echo Starting Firefox
         firefox localhost:9876
       ) &
-      bin/ci/docker-run.sh -p 9876:9876 node bin/ci/karma-ci.js
+      bin/ci/docker-run.sh -e NODE_ENV=production -p 9876:9876 node bin/ci/karma-ci.js
```

Comment:
This doesn't have any effect because the karma tests use a different webpack config than the main one (I still see proptype errors in my test runs). Which is a separate bug. We'll want this in the long term though.